dmoj/cptbox/landlock_helpers.cpp

```diff
@@ -0,0 +1,37 @@
+#ifndef __FreeBSD__
+
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "landlock_helpers.h"
+
+int landlock_add_rules(const int ruleset_fd, const char *const *const paths,
```

Comment:
`const char **paths` would have been fine. The amount of `const`s you are adding is excessive.